app/views/questions/_fields.html.erb

```diff
@@ -9,32 +9,31 @@
 
   <%= f.base_errors %>
 
-  <%= f.field(:code, :required => true, :maxlength => 20, :read_only => cannot?(:update_code, @question)) %>
+  <%= f.field(:code, required: true, maxlength: 20, read_only: cannot?(:update_code, @question)) %>
 
   <%# core fields should not be editable if don't have update_core permission %>
   <% cant_update_core = cannot?(:update_core, @question) %>
 
-  <%= f.field(:qtype_name, :type => :select, :required => true, :read_only => cant_update_core,
-        :options => translate_options(@question_types.map(&:name), :question_type)) %>
+  <%= f.field(:qtype_name, type: :select, required: true, read_only: cant_update_core,
+        options: translate_options(@question_types.map(&:name), :question_type)) %>
 
   <%# this is a partial because it contains more complex controls %>
   <%# the partial path is fully qualified because THIS partial might be included in several different contexts %>
-  <%= f.field(:option_set_id, :partial => 'questions/option_set_select', :read_only => cant_update_core) %>
+  <%= f.field(:option_set_id, partial: 'questions/option_set_select', read_only: cant_update_core) %>
 
   <%# title and hint boxes, one for each language %>
   <div class="names_hints">
     <% configatron.preferred_locales.each_with_index do |lang, rank| %>
-      <%= render("questions/title_and_hint", :f => f, :lang => lang, :rank => rank) %>
+      <%= render("questions/title_and_hint", f: f, lang: lang, rank: rank) %>
     <% end %>
   </div>
 
-  <div>
-    <%= f.field(:audio_prompt, type: :file) %>
-  </div>
+  <%= f.field(:reference, type: :text) %>
+  <%= f.field(:audio_prompt, type: :file) %>
 
   <div class="minmax">
-    <%= f.field(:casted_minimum, :partial => 'questions/min_max_condition') %>
-    <%= f.field(:casted_maximum, :partial => 'questions/min_max_condition') %>
+    <%= f.field(:casted_minimum, partial: 'questions/min_max_condition') %>
+    <%= f.field(:casted_maximum, partial: 'questions/min_max_condition') %>
```

Comment:
Yay cleanups!

Reply:
Can you also change single quotes to double?

Reply:
done